core/domain/user_jobs_continuous_test.py

```diff
@@ -929,102 +766,41 @@ def test_scaler_multiplier_independence(self):
         self.assertEqual(
             user_stats_model.impact_score, expected_user_impact_score)
 
-    def test_no_ratings_independence(self):
-        """Test that when one exploration has no ratings, the other
-        exploration's impact score is not impacted.
-        """
-        # Sign up a user and have them create two explorations.
+    def test_only_yield_when_rating_greater_than_two(self):
+        """Tests that map only yields an impact score for an
+        exploration when the impact score is greater than 0."""
+        # Sign up a user and have them create an exploration.
         user_a_id = self._sign_up_user(
             self.USER_A_EMAIL, self.USER_A_USERNAME)
-        exploration_1 = self._create_exploration(self.EXP_ID_1, user_a_id)
-        exploration_2 = self._create_exploration(self.EXP_ID_2, user_a_id)
-        # Give one exploration as many ratings as necessary to avoid
-        # the scaler for number of ratings. Don't give the other any ratings.
-        self._rate_exploration(
-            exploration_1.id, self.NUM_RATINGS_SCALER_CUTOFF,
-            self.ABOVE_MIN_RATING)
-        # Give these explorations more than one playthrough (so
-        # ln(num_completions) != 0.
-        self._complete_exploration(exploration_1, self.MIN_NUM_COMPLETIONS)
-        self._complete_exploration(exploration_2, self.MIN_NUM_COMPLETIONS)
-        # We expect the second exploration to yield 0 (since it has no
-        # ratings), so the expected impact score is just the impact score for
-        # exploration 1.
-        expected_user_impact_score = round(
-            math.log(self.MIN_NUM_COMPLETIONS) *
-            (self.ABOVE_MIN_RATING - self.MIN_AVERAGE_RATING) *
-            self.MULTIPLIER)
-        # Verify that the impact score matches the expected.
+        self._create_exploration(self.EXP_ID_1, user_a_id)
+        self._rate_exploration(self.EXP_ID_1, 2, 1)
+
         self._run_computation()
-        user_stats_model = user_models.UserStatsModel.get(user_a_id)
-        self.assertEqual(
-            user_stats_model.impact_score, expected_user_impact_score)
+        user_stats_model = user_models.UserStatsModel.get(user_a_id,
+                                                          strict=False)
+        self.assertEqual(user_stats_model, None)
```

Comment:
Actually I'm trying this, but the way our tests are written, it isn't allowing another test to run while the other test is "running"....

Exception: Attempted to start computation ModifiedUserImpactAggregator, which is already running.

Not sure how to get around that....I'll run the other tests without those changes for now and see what I can do about that afterward.